server/world/world.go

```diff
@@ -1451,6 +1454,7 @@ func (w *World) addViewer(c *chunkData, viewer Viewer) {
 	}
 
 	viewer.ViewTime(w.Time())
+	viewer.ViewWorldSpawn(w.Spawn())
```

Comment:
You might want to move this to the end of `(*World*).addWorldViewer()`. The `ViewTime` call is actually misplaced too, as this function is called for every chunk loaded. Would you mind moving that to the same function too?
Thanks!

Reply:
Done!